taxi/views.py

```diff
@@ -2,14 +2,16 @@
 from django.shortcuts import render
 from django.views import generic
 from django.contrib.auth.mixins import LoginRequiredMixin
```

Comment:
The `LoginRequiredMixin` import is not grouped correctly. According to the checklist, you should group imports using `()` if needed. Please, correct it.